.github/workflows/TestBinaryInstaller.yml

```diff
@@ -58,6 +58,8 @@ jobs:
           ddev exec --raw composer config minimum-stability dev
           ddev composer require "lullabot/drainpipe @dev" --with-all-dependencies
           ddev composer require "lullabot/drainpipe-dev @dev" --dev --with-all-dependencies
+          # Ensure scaffold files are properly copied
+          ddev exec --raw composer dump-autoload
```

Comment:
Interesting, did something fail for you at some point?

The autoloader is updated when you run `composer require` so I don't think actually adds anything.

Reply:
@beto-aveiga We can remove this line after we get all other checks passing.